### DISCUSSION TITLE
Adding an issue from an unrelated org/repo to my project

### DISCUSSION THREAD

**Discussion by thibaudcolas** [community/community#69763] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

I’m able to add an issue to my project that’s from a completely separate repository/organization which I don’t have access to. I’m not entirely sure if this is a bug or a cool feature?

Here is the link to the issue within the project for reference: https://github.com/orgs/wagtail/projects/35?pane=issue&itemId=41207834

I was able to add it to my project by pasting the issue’s URL in the project’s "Add" input.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by rileybroughten** [comment, marked as answer]
👋 You can search for items using `#` to search for an organization or repository. There are more details in the [changelog announcement](https://github.blog/changelog/2023-02-23-github-issues-projects-february-23rd-update/#add-cross-organization-issues-and-pull-requests-to-projects).

Let me know if this helps!

**Reply by thibaudcolas** [reply]
Thank you @rileybroughten, it does :)

**Reply by Julian-Dumitrascu** [reply]
👋 I like your messages. You make me like this service more than others.
@thibaudcolas I'm glad when somebody is provided a response, especially if it's sufficient. When it comes so soon, it's even better.

**Comment by abdul-musabbir** [comment]
Thank you
